src/bindings/interface/logger.i

```diff
@@ -10,7 +10,7 @@
 %feature("docstring") libcellml::Logger::addError
 "Adds an error to this logger.";
 
-%feature("docstring") libcellml::Logger::clearErrors
+%feature("docstring") libcellml::Logger::removeAllErrors
```

Comment:
Qhat's the rationale behind `removeAllErrors` - are we able to remove them selectively eg with a `removeError(index)` function?  That could be useful if/when we get to having different levels or types of error.  If not, I would prefer it just be `removeErrors`

Reply:
The rationale is that when we are removing all types of thing from an object we have used `removeAll*` to do that job.

Reply:
There is currently no functionality to remove/take selective errors.

Reply:
Ok, thanks.